frontend/src/old-pages/Configure/SlurmMemorySettings.tsx

```diff
@@ -0,0 +1,119 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+// with the License. A copy of the License is located at
+//
+// http://aws.amazon.com/apache2.0/
+//
+// or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+// OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as React from 'react'
+import {Trans, useTranslation} from 'react-i18next'
+import {
+  Container,
+  Header,
+  Alert,
+  Toggle,
+  SpaceBetween,
+  Popover,
+} from '@awsui/components-react'
+import {setState, getState, useState, clearState} from '../../store'
+import styles from './css/SlurmMemorySettings.module.css'
+
+function SlurmMemorySettings() {
+  const {t} = useTranslation()
+  const [infoAlertVisible, setInfoAlertVisible] = React.useState(false)
+  const slurmSettingsPath = [
+    'app',
+    'wizard',
+    'config',
+    'Scheduling',
+    'SlurmSettings',
+  ]
+  const memoryBasedSchedulingEnabledPath = [
+    ...slurmSettingsPath,
+    'EnableMemoryBasedScheduling',
+  ]
+  const memoryBasedSchedulingEnabled = useState(
+    memoryBasedSchedulingEnabledPath,
+  )
+
+  const toggleMemoryBasedSchedulingEnabled = () => {
+    const setMemoryBasedSchedulingEnabled = !memoryBasedSchedulingEnabled
+    if (setMemoryBasedSchedulingEnabled)
+      setState(
+        memoryBasedSchedulingEnabledPath,
+        setMemoryBasedSchedulingEnabled,
+      )
+    else {
+      clearState(memoryBasedSchedulingEnabledPath)
+      if (Object.keys(getState([...slurmSettingsPath])).length === 0)
+        clearState([...slurmSettingsPath])
+    }
+  }
+
+  return (
+    <Container
+      header={
+        <Header variant="h2">
+          <SpaceBetween size="xs" direction="horizontal">
+            {t('wizard.queues.slurmMemorySettings.container.title')}
+            <Popover
+              dismissButton={true}
+              position="right"
+              size="small"
+              triggerType="custom"
+              content={
+                <Trans i18nKey="wizard.queues.slurmMemorySettings.container.help">
+                  <a
+                    rel="noopener noreferrer"
+                    target="_blank"
+                    href="https://slurm.schedmd.com/cgroup.conf.html#OPT_ConstrainRAMSpace"
+                  ></a>
+                </Trans>
+              }
+            >
+              <a href="#" className={styles.info}>
+                {t('wizard.queues.slurmMemorySettings.container.info')}
+              </a>
```

Comment:
We could simply leverage the provided [Info link](https://polaris.a2z.com/components/link/?tabId=playground&example=info-link)

It's not recommended to use them to trigger popovers ([source](https://polaris.a2z.com/components/popover/?tabId=usage)), but for the time being this is the solution we should stick with (we'll implement the help panel in the future)

In general I'd avoid using custom implementations, but rather ping our designer of reference with a suggestion taken from the design system